src/openrct2/common.h

```diff
@@ -23,10 +23,13 @@
 #include <cassert>
 #include <cstddef>
 #include <cstdint>
+#include <string>
 
 using utf8 = char;
 using utf8string = utf8*;
 using const_utf8string = const utf8*;
+using u8string = std::basic_string<utf8>;
+using u8string_view = std::basic_string_view<utf8>;
```

Comment:
I would prefer this entire section to be rather in String.hpp, the plan is to get rid of common.h so everything is forced to include the things it needs.